transcribed/page0010.txt

```diff
@@ -1,56 +1,56 @@
-001             TITLE ITS 1.0 3/19/67                                           PAGE 1
-002             MOBY==1
+001              TITLE ITS 1.0 3/19/67                                                 PAGE 1
+002              MOBY== 1
 003
-004             ;AC DEFS
```

Comment:
This must be a semicolon introducing a comment.  A colon isn't valid MIDAS syntax.

Reply:
you are right! :)